test/jbuild

```diff
@@ -0,0 +1,11 @@
+(executable
+  ((name rrdp_dummy)
+   (libraries (rrdd-plugin
+               xenstore.unix
+               xenstore_transport.unix))
+  ))
+
+(alias
+ ((name    runtest)
+  (deps    (rrdp_dummy.exe))
+  (action  (run ${<}))))
```

Comment:
Hmm, so we do have a test? Could we run this from the opam file?

Reply:
No, it's explained in the PR description:

> Tests are disabled because they need to run on a xenserver host

Reply:
You can build or copy them on a xenserver host and run them.
```
$ _build/default/test/rrdp_dummy.exe --help
Usage: /home/marcellos/projects/jbuilder/ocaml-rrdd-plugin/_build/default/test/rrdp_dummy.exe -mode [local|interdomain]
  -mode Switch between local and interdomain mode
  -help  Display this list of options
  --help  Display this list of options
```
Rnning them locally you will get xen and unix related failures

Reply:
On the host you get something like:
```
./rrdp_dummy.exe
[20171213T13:52:58.000Z| info|dt21|0 ||xcp-rrdd-dummy] Obtained path=/dev/shm/metrics/xcp-rrdd-dummy

```

Reply:
We should set TESTS to false for the Travis build then.

Reply:
I'll do that... anyways there is no `build-test` in `opam` so nothing will run